src/content/docs/browser/browser-integrations/wordpress-fullstack-integration.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: WordPress monitoring integration
+tags:
+    - New Relic integrations
+    - WordPress Full Stack integration
+metaDescription: Use New Relic browser monitoring to get a dashboard with metrics from your Wordpress.
+---
+
+
+import infrastructureWordPressIntegrationDashboard from 'images/infrastructure_screenshot-full_wordpress-dashboard.webp'
+
+Our WordPress integration monitors the performance of your WordPress web application, helping you diagnose issues in your application and optimize your code. Our WordPress integration makes use of our PHP, Apache, and MySQLmonitoring agents, and gives you a pre-built dashboard with your most important WordPress metrics like transactions, visitors and call duration.
+
+<img
+  title="WordPress dashboard"
+  alt="A screenshot depicting the wordpress dashboard"
+  src={infrastructureWordPressIntegrationDashboard}
+/>
+
+<figcaption>
+After setting up our WordPress integration, we give you a dashboard for your WordPress metrics. 
+</figcaption>
+
+<Steps>
+<Step>
+### Install the relevant agents[#infra-install]
+
+1.  Follow [the infrastructure agent install steps](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent) for the host containing your WordPress app.
+2. Open [PHP quickstart installation](https://newrelic.com/instant-observability/php).
+Click on install button to start the PHP agent installation.
+3. Open [MySQL quickstart installation](https://newrelic.com/instant-observability/mysql).
+Click on install button to start the MySQL agent installation.
```

Comment:
we have a lot of non-sentences in here, be good to review. e.g., "Click on install button" and "will automatically install flex binary"